requests/cookies.py

```diff
@@ -13,6 +13,8 @@
 import time
 import calendar
 import collections
+
+from . import utils
```

Comment:
Yup, so this isn't going to work. This creates a circular import that breaks the code.

My recommendation is that we create an `_internal_utils.py` module that defines `to_native_string`. We then import from there in both this module and `utils.py` to expose the name in both places. How does that sound?